plugin/error_vis.py

```diff
@@ -184,6 +218,63 @@ def _as_html(errors_dict):
         return errors_html
 
     @staticmethod
+    def _as_phantom_html(errors_dict):
+        """Get error as html for phantom.
+
+        Args:
+            errors_dict (dict): Current error
+        """
+        stylesheet = '''
+            <style>
```

Comment:
I would put all text transformations (it is done already?) and this html stuff in one place.